packages/rmf-auth/package.json

```diff
@@ -21,41 +22,49 @@
   "author": "matiasbavera@gmail.com",
   "license": "Apache-2.0",
   "dependencies": {
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react-hooks": "^3.4.2",
-    "@testing-library/user-event": "^12.1.9",
-    "@types/node": "12.12.14",
-    "@types/react": "^16.9.49",
-    "@types/react-router": "^5.1.7",
-    "@types/react-router-dom": "^5.1.5",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.56",
     "eventemitter3": "^4.0.7",
     "keycloak-js": "^11.0.2",
-    "react-router": "^5.2.0",
     "styled-components": "^4.4.1",
     "typeface-roboto": "0.0.75"
   },
   "devDependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@storybook/addon-actions": "^6.2.9",
+    "@storybook/addon-essentials": "^6.2.9",
+    "@storybook/addon-links": "^6.2.9",
+    "@storybook/react": "^6.2.9",
+    "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.2.3",
+    "@testing-library/react-hooks": "^5.0.3",
+    "@testing-library/user-event": "^12.1.9",
+    "@types/debug": "^4.1.5",
     "@types/jest": "^26.0.13",
-    "@types/react-dom": "^16.9.8",
+    "@types/react": "^16.9.49",
+    "@types/react-dom": "^16.9.12",
+    "@types/react-router": "^5.1.7",
+    "@types/react-router-dom": "^5.1.7",
     "concurrently": "^5.3.0",
-    "@types/debug": "^4.1.5",
+    "eslint": "^7.26.0",
+    "eslint-plugin-react": "^7.21.4",
+    "eslint-plugin-react-hooks": "^4.1.2",
     "gh-pages": "^3.1.0",
-    "react-test-renderer": "^17.0.1",
-    "ts-node": "^9.0.0",
     "jest": "^26.5.0",
+    "react": "^16.13.1",
```

Comment:
That is correct, one is a peer dependency which will not be automatically installed in npm < 7, the other is a dev dependency because we need to install it to run the tests.